src/njoy21/input/VIEWR/Card1/Nps.hpp

```diff
@@ -0,0 +1,28 @@
+struct Nps { 
```

Comment:
When I do that, I'm getting erros, and I have a guess as to why. Since the verify function in `argument::common::Nout `is expecting the second parameter to be type Nin, it's getting upset that what I pass in in my test cases is of type Infile... does that sound possible?